types/recharts/index.d.ts

```diff
@@ -86,7 +87,7 @@ export interface BarProps {
 	name?: string | number;
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?